### PR TITLE
Azure CLI 2021-02 beta release

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

File: docs-ref-conceptual/install-azure-cli-beta.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+      ### [Windows PowerShell](#tab/powershell)
+
+   ```powershell
+   . .\$env\Scripts\Activate.ps1
+   ```
+
+   ### [Linux/macOS Bash](#tab/bash)
+
+   ```console
```
Comment (jiasli) [inline docs-ref-conceptual/install-azure-cli-beta.md:72]: Should we use `console` even we have identified it is Linux Bash?

File: docs-ref-conceptual/install-azure-cli-beta.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+   . $env/bin/activate
+   ```
+   ---
+   These commands can also be used to reactivate your virtual environment.
+
+1. Install Azure CLI beta
+
+   ```console
+   pip install --pre --extra-index-url https://azurecliprod.blob.core.windows.net/beta/simple/ azure-cli
```
Comment (jiasli) [inline docs-ref-conceptual/install-azure-cli-beta.md:81]:
```suggestion
   pip install --pre --extra-index-url https://az.blob.core.windows.net/beta/simple/ azure-cli
```

File: docs-ref-conceptual/install-azure-cli-beta.md (hunk 1 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/install-azure-cli-beta.md:72]: @jiasli , writers twitch between the two.  I'm happy to make these "bash" vs "console".

File: docs-ref-conceptual/install-azure-cli-beta.md (hunk 2 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/install-azure-cli-beta.md:81]: @jiasli, then in "install-azure-cli-windows.md" this 2nd URL is also correct -->

In order to get the MSI, your proxy needs to allow HTTPS connections to the following addresses:

* `https://aka.ms/`
* `https://azcliprod.blob.core.windows.net/`

File: docs-ref-conceptual/install-azure-cli-windows.md [hunk 3]
```diff
@@ -99,7 +88,7 @@ or for situations where your machine may be configuration-managed or require adv
 In order to get the MSI, your proxy needs to allow HTTPS connections to the following addresses:
 
 * `https://aka.ms/`
-* `https://azcliprod.blob.core.windows.net/`
+* `https://azurecliprod.blob.core.windows.net/`
```
Comment (jiasli) [inline docs-ref-conceptual/install-azure-cli-windows.md]: I think we need to keep the existing `azcliprod`.

File: docs-ref-conceptual/install-azure-cli-beta.md (hunk 2 above)
Comment (fengzhou-msft) [reply docs-ref-conceptual/install-azure-cli-beta.md]: Yes. The MSI is already migrated to `azcliprod`.

**Comment by dbradish-microsoft** [comment]
@jiasli , This PR is ready to go.  Please provide date to merge.

**Review by jiasli — APPROVED** [review]

**Pull request merged**